lib_nbgl/include/nbgl_touch.h

```diff
@@ -23,18 +23,19 @@ extern "C" {
 // duration of a short touch on touch panel (in ms)
 #define SHORT_TOUCH_DURATION 0
 // duration of a long touch on touch panel (in ms)
-#define LONG_TOUCH_DURATION  1500
+#define LONG_TOUCH_DURATION  3000
 /**********************
  *      TYPEDEFS
  **********************/
 
 /**********************
  * GLOBAL PROTOTYPES
  **********************/
-void        nbgl_touchHandler(nbgl_touchStatePosition_t *touchEvent, uint32_t currentTimeMs);
-bool        nbgl_touchGetTouchedPosition(nbgl_obj_t                 *obj,
-                                         nbgl_touchStatePosition_t **firstPos,
-                                         nbgl_touchStatePosition_t **lastPos);
+void nbgl_touchInit(bool fromUx);
+void nbgl_touchHandler(bool fromUx, nbgl_touchStatePosition_t *touchEvent, uint32_t currentTimeMs);
+bool nbgl_touchGetTouchedPosition(nbgl_obj_t                 *obj,
+                                  nbgl_touchStatePosition_t **firstPos,
+                                  nbgl_touchStatePosition_t **lastPos);
```

Comment:
Why do we need to have separate contexts for ux and apps ?

Reply:
As explained in the architecture document, UX and App are now sharing many variables, but for the communication between App and UX (event forwarding and ability to use events in App), it's sometime necessary to know the number of screens consumed by UX, or, for Touch management, to have separated contexts, to compute NBGL Touch Events (touch duration for example)